### ISSUE TITLE
Term Request: Fingerprint-like corneal lines

### ISSUE THREAD

**Issue by daboe01** [obophenotype/human-phenotype-ontology#11738]
**Preferred term label:**
Fingerprint-like corneal lines

**Synonyms**
* Corneal fingerprint lines
* Curvilinear intraepithelial corneal ridges
* Fingerprint lines of cornea

**Definition (free text, please give PubMed ID)**
Fine, parallel, curvilinear intraepithelial lines in the paracentral or central cornea, resembling the concentric contours of a fingerprint, best visualized on retroillumination. Ultrastructurally, they represent rib-like intraepithelial extensions and folds of abnormal basement membrane material (PMID: 38359414).

**Parent term (use hpo.jax.org/app)**
Abnormal corneal epithelium morphology (HP:0011495)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
* Epithelial basement membrane dystrophy (EBMD) (ORPHA:98956, OMIM:121820)

**Your nano-attribution (ORCID)**
0000-0001-9042-0508
